### DISCUSSION TITLE
PyData Global 2021

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#286] [Show and tell]
I'm giving a talk at PyData Global about using Piccolo. These are the draft slides.

[pydata_global_presentation.pdf](https://github.com/piccolo-orm/piccolo/files/7272914/pydata_global_presentation.pdf)

I'm mostly going to demo Piccolo Admin, as it's a great tool for managing data, and hopefully this talk will make more data scientists aware of it.

I'll post the video here when it becomes available online (could be a while though).

**Comment by sinisaos** [comment]
@dantownsend Great. Good luck.

**Reply by dantownsend** [reply]
Thanks!

**Comment by dantownsend** [comment]
I've done the presentation now - these are the [final slides](https://github.com/piccolo-orm/pymdb/blob/master/pydata_global_presentation.pdf).

I demoed this example [Piccolo app](https://github.com/piccolo-orm/pymdb/). Seemed to go well.

The video will be available on YouTube in 2 months ...

**Reply by AliSayyah** [reply]
That's great. can't wait to see it!

**Comment by yezz123** [comment]
Great Talk @dantownsend, I just give one relate to FastAPI and Farm Stack yesterday at A big Event in the Moroccan Community.
Wish you a good luck in your talk 🚀✨

**Reply by dantownsend** [reply]
@yezz123 Thanks very much! Congrats on your talk, it sounds great.

**Comment by dantownsend** [comment]
It's online now - https://www.youtube.com/watch?v=aRmrXf_zOfQ
